src/components/List/EmptyList.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import classnames from "classnames"
+
+interface EmptyListProps {
```

Comment:
This should be `ListEmptyProps` because Misago uses Facebook's convention for naming components.